sdk/identity/azure-identity/azure/identity/aio/_credentials/environment.py

```diff
@@ -77,5 +72,6 @@ def __init__(self, **kwargs: "Any") -> None:
         :raises ~azure.identity.CredentialUnavailableError: environment variable configuration is incomplete
         """
         if not self._credential:
-            raise CredentialUnavailableError(message=self._unavailable_message)
+            raise CredentialUnavailableError(message="EnvironmentCredential authentication unavailable. "
```

Comment:
You can use this syntax to fit the message on one line:
```suggestion
            message = (
                "EnvironmentCredential authentication unavailable. Environment variables are not fully configured."
            )
            raise CredentialUnavailableError(message=message)
```
(applies similarly to other instances)